first_last_position_sorted_array.py

```diff
@@ -0,0 +1,57 @@
+# The logic bhind solving this problem is binary search modification. As taught in class, I need to do 2 binary searches to find lftmost and right most index.
```

Comment:
Good work on the comments, Please make them shorter and well organized.
Your comments should be an overview of your approach and not a detailed account.